models/Movie.model.js

```diff
@@ -0,0 +1,18 @@
+const { mongoose, Schema, model } = require('mongoose');
+
+const movieSchema = new Schema(
+    {
+        title: String,
+        genre: String,
+        plot: String,
+        cast: [{
+            type: mongoose.Schema.Types.ObjectId,
```

Comment:
If you keep the way above you could use the previous one or the following one

```suggestion
            type: Schema.Types.ObjectId,
```